test/net/sourceforge/kolmafia/textui/ParserTest.java

```diff
@@ -1268,6 +1268,18 @@
 				"Cannot initialize parameter x",
```

Comment:
When _are_ we supposed to be able to initialize parameters?

All the tests exercise failures, but not a single success.

Reply:
Never. This is, again, preparation for the "keep parsing until the end of the file, and accumulate errors". We parse the initialization even though we know it's not allowed, so that we can tell when it *ends*, and the next parameter starts.

Additionally, this could lay the groundwork for allowing "default values" to parameters... But that's just a side-effect.